source/dlangbot/github.d

```diff
@@ -27,22 +27,17 @@ string formatComment(in ref PullRequest pr, in IssueRef[] refs, in Issue[] descs
 
     auto app = appender!string();
     app.formattedWrite(
-`Thanks for your pull request, @%s!  We are looking
```

Comment:
This is a WYSIWYG string, so this won't work, e.g.

![image](https://user-images.githubusercontent.com/4370550/27476915-db9f410c-580a-11e7-8283-c9a294c91a90.png)